### PR TITLE
[Dropdown]: Various usability improvements & bugfixes

### PR DESCRIPTION
**Pull request by fallaciousreasoning**

### CODE REVIEW

File: examples/plain-html/index.html [hunk 1]
```diff
@@ -230,59 +257,61 @@
...
-        </div>
-        <leo-hr></leo-hr>
-        <div class="navigation-action-items">
-          <leo-navigationitem outsideList="true" icon="help-outline">Help</leo-navigationitem>
-          <leo-navigationitem outsideList="true" icon="outside">Log out</leo-navigationitem>
-        </div>
-      </leo-navigationactions>
-    </leo-navigation>
-  </div>
+      <leo-navigation>
```
Comment (fallaciousreasoning) [inline examples/plain-html/index.html:260]: This is just formatting

**Comment by fallaciousreasoning** [comment]
@AlanBreck could you take a look at this? (the change is much smaller than it looks, but it formatted a bunch of lines in index.html)

File: src/components/formItem/formItem.svelte [hunk 2]
```diff
@@ -211,6 +212,10 @@
     display: flex;
     flex-direction: row;
     gap: var(--leo-spacing-s);
+
+    &:empty {
+      display: none;
+    }
```
Comment (AlanBreck) [inline src/components/formItem/formItem.svelte:218]: Brilliant!!!

File: src/components/formItem/formItem.svelte [hunk 3]
```diff
@@ -135,10 +134,12 @@
 
       & .container:has(*:focus-visible),
       &.isFocused .container {
-        color: var(--color-focus);
-        background: var(--background-focus);
-        box-shadow: var(--shadow-focus);
-        border-color: var(--border-color-focus);
+        &:not(:has(.extra-content:focus-within)) {
```
Comment (AlanBreck) [inline src/components/formItem/formItem.svelte:137]: What's going on here?

**Review by AlanBreck — APPROVED** [review]

File: src/components/formItem/formItem.svelte (hunk 3 above)
Comment (fallaciousreasoning) [reply src/components/formItem/formItem.svelte:137]: Only show the focus border if the focus is inside the input element, not within the .extra content element.

**Pull request merged**